.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
It's good practice to have newlines at the end of files. There is a VSCode setting to do that automatically on save. Otherwise, you can see this icon on Pull Requests:
![image](https://github.com/istocoana/RoRinternshipWD-A1/assets/19929642/10475d73-a792-44ac-89b3-dfc39be37d48)